harper-core/src/parsers/mod.rs

```diff
@@ -1,14 +1,23 @@
+mod cases;
 mod markdown;
 mod mask;
 mod plain_english;
 
 use blanket::blanket;
+pub use cases::Cases;
 pub use markdown::Markdown;
 pub use mask::Mask;
 pub use plain_english::PlainEnglish;
 
 pub use crate::token::{Token, TokenKind, TokenStringExt};
 
+#[cfg(not(feature = "concurrent"))]
```

Comment:
I'm surprised this wasn't already here. How was `harper-cli` able to compile?

Reply:
Idk, I just checked, and it turns out `mask::Masker` uses Sync too and doesn't have a concurrent cfg.